src/pdf_beamtime/include/pdf_beamtime/simple_server.hpp

```diff
@@ -0,0 +1,59 @@
+#ifndef SIMPLE_SERVER_HPP
+#define SIMPLE_SERVER_HPP
+
+#include <chrono>
+#include <fstream>
+#include <iostream>
+#include <functional>
+#include <memory>
+#include <thread>
+
+#include <rclcpp/rclcpp.hpp>
+#include <rclcpp_action/rclcpp_action.hpp>
+
+#include <moveit/move_group_interface/move_group_interface.h>
+#include <moveit/planning_scene_interface/planning_scene_interface.h>
+#include <hello_moveit_interfaces/action/simple_action_msg.hpp>
+
+/// @brief Create the obstacle environment and an simple action server for the robot to move
+class SimpleServer
+{
+public:
+  using SimpleActionMsg = hello_moveit_interfaces::action::SimpleActionMsg;
+
+  SimpleServer(const std::string & move_group_name, const rclcpp::NodeOptions & options);
```

Comment:
make this explicit. Not sure if that keyword goes in the header or in the source.